### DISCUSSION TITLE
cant find how to open it

### DISCUSSION THREAD

**Discussion by ArsenalTheProtogen** [DigitalLogicSimCommunity/Digital-Logic-Sim-CE#84] [Q&A]
where do i open

**Comment by AOx0** [comment]
Hi.

I suppose you are using windows? In that case, just unzip the object for windows available at the [latest release](https://github.com/DigitalLogicSimCommunity/Digital-Logic-Sim-CE/releases/tag/v0.39.1), probably you want the `x86_64` object.

Then execute the file named `Digital Logic Sim.exe` that is inside the decompressed folder.

**Reply by ArsenalTheProtogen** [reply]
yeah the file for some reason doesnt exist, and i cant find it

**Reply by AOx0** [reply]
It doesn't? The file structure of the decompressed file should look like:

- Digital Logic Sim/
    - Digital Logic Sim.ese
    - Digital Logic Sim_Data/
        - ...
    - MonoBleedingEdge
        - ...
    - UnityCrashHandler.exe
    - UnityPlayer.dll

**Comment by ArsenalTheProtogen** [comment]
![image](https://github.com/DigitalLogicSimCommunity/Digital-Logic-Sim-CE/assets/119634418/d9d2eef6-03c0-4a06-88a8-823476eea1af)
![image](https://github.com/DigitalLogicSimCommunity/Digital-Logic-Sim-CE/assets/119634418/1a0436f8-1795-4ab9-ae67-c35697f167ba)
this is all that i see

**Reply by AOx0** [reply]
Oh, I see, that is the source code of the project, you can compile it using Unity. Or you can download one of the already compiled objects at the [latest release](https://github.com/DigitalLogicSimCommunity/Digital-Logic-Sim-CE/releases/tag/v0.39.1)

**Reply by ArsenalTheProtogen** [reply]
oh ok thanks

**Reply by AOx0** [reply]
You probably want this one:

![image](https://github.com/DigitalLogicSimCommunity/Digital-Logic-Sim-CE/assets/50227494/804b2924-ec9c-44bb-a585-72ad568ed732)

**Reply by ArsenalTheProtogen** [reply]
i got it, thanks!
